<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpSenderGUI</class>
 <widget class="QDialog" name="UdpSenderGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TerraME Observer :: UDP Sender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>UDP Log messages: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="logEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPortStatus">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>18</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Sending at: 456456</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCompress">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>18</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Compress off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblStatesSent">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>18</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>States sent: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMessagesSent">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>18</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Messages sent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSpeedStatus">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>18</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Speed: unknown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>UdpSenderGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
